runtime-modules/proposals/codex/src/weights.rs

```diff
@@ -191,13 +192,13 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ProposalEngine Proposals (r:0 w:1)
 	// Storage: ProposalEngine DispatchableCallCode (r:0 w:1)
 	fn create_proposal_create_working_group_lead_opening(i: u32, t: u32, d: u32, ) -> Weight {
-		(113_645_000 as Weight)
-			// Standard Error: 11_000
-			.saturating_add((1_542_000 as Weight).saturating_mul(i as Weight))
-			// Standard Error: 62_000
-			.saturating_add((504_000 as Weight).saturating_mul(t as Weight))
-			// Standard Error: 62_000
-			.saturating_add((618_000 as Weight).saturating_mul(d as Weight))
+		(112_039_000 as Weight)
+			// Standard Error: 5_000
+			.saturating_add((1_387_000 as Weight).saturating_mul(i as Weight))
+			// Standard Error: 44_000
+			.saturating_add((1_000 as Weight).saturating_mul(t as Weight))
```

Comment:
This looks wrong, but probably it was just some interference during weight function generation.